src/Handlers/Application/ContainerHandler.php

```diff
@@ -2,8 +2,6 @@
 
 namespace Psalm\LaravelPlugin\Handlers\Application;
 
-use Illuminate\Foundation\AliasLoader;
-use Illuminate\Support\Facades\RateLimiter;
```

Comment:
@mr-feek When I noticed this unsed import, I wondered if this line is/was causing #134. 🤔